### ISSUE TITLE
Players Fuel

### ISSUE THREAD

**Issue by KaMiKaZeE1221** [my-name-is-samael/BeamJoy#20]
I think we need something like an emergency refuel or stopping the fuel from going down any further than 0.2 liters if fuel (when the car starts sputtering and carrying on) 

As of right now with fuel persistence nothing stops you if you forget to refuel from just running out of fuel and then needing to spawn a new vehicle.

I think this would be a nice feature to have.

**Comment by my-name-is-samael** [comment]
Good idea, having a button appearing with very low fuel. Clicking it would trigger the refuel animation but longer (configurable)

**Comment by my-name-is-samael** [comment]
Feature will include config windows (Freeroam & Server) autoclose en group demote/permission loss
